### ISSUE TITLE
Add DONATE button to restic repo

### ISSUE THREAD

**Issue by yhafri** [restic/restic#884]
As i shared couple of times, i'd like to say thanks to `restic`, the community around it and would like to donate.

For that, I would like to see a visible `DONATE` button somewhere [here](https://github.com/restic/restic).

**Comment by fd0** [comment]
Hm, I need to think about this. Let me get back to you.

**Comment by TamalSaha** [comment]
@fd0, we would also like to hear what are your plans for restic? Do you have any commercial plans?

**Comment by fd0** [comment]
I don't have any commercial plans. I started restic to scratch my own itch.

**Comment by aut0** [comment]
Please add a donate button

**Comment by fd0** [comment]
You could also send me plush Gophers, I still need the light blue (dotGo 2017 edition) and the [yellow/green one](https://twitter.com/francesc/status/903679109229821952). That'd be nice :)

**Comment by JackBlackmore** [comment]
resurrecting an old issue but are you sure you don't want to add a donate button? :) After playing around with a few backup solutions previously this has turned out to be EXACTLY what I was after and the amount of time you seem to be putting in answering issues alone deserves admiration! 

Thank you so much for this man, it's massively appreciated!

**Comment by fd0** [comment]
To the people subscribed to this issue: While it has nothing to do with restic directly, it would mean a lot to me personally if some of you would donate to the [fundraiser of Matt Holt](https://twitter.com/mholt6/status/948687667264307200) which would allow him (a college student) to speak at a conference half way around the world. He's not only a restic contributor, but also one of the authors of the [Caddy web server](https://caddyserver.com)!

**Comment by awehrfritz** [comment]
@fd0: have you given this some further thought? I strongly believe that you arrange for a way to be rewarded for your tremendous effort. You could place something like this right on the front page of restic.net, similarly to what the guys at cryptomator.org do. If you feel that this is to much, then at least consider adding an option somewhere else (maybe in the contribute section) that is a bit more hidden.

Either way, your work on restic is very much appreciated - this is by far the best backup utility I have come across. I believe there would be many people (including me) that would happily donate a few $ as a token of appreciation for your time and effort.

**Comment by awehrfritz** [comment]
In light of the recently introduce Github Sponsors program, I just thought to mention this article about the matching funds program here:
https://help.github.com/en/articles/about-github-sponsors#about-the-github-sponsors-matching-fund

**Comment by extragoofy** [comment]
I'm resurrecting this again - restic is genious and works really well. I'd really like to show my appreciation somehow. Thank you so much.

**Comment by rawtaz** [comment]
Thanks to everyone for their lovely comments and interesting in giving back. It's the words that count most of all!

Unfortunately I'll close this now, as there's still no way to donate to restic or @fd0 or similar. It's been discussed just now to make sure I'm not doing the wrong thing here, but restic just isn't open for donations. I'm sorry :)

If you want to give back to restic the project, feel free to help other users when they have problems - this way core developers and maintainers can spend more time on other matters to keep restic running. No expectations though, we're just happy to have you around regardless!

**Comment by molecular** [comment]
@rawtaz , @fd0 

Was looking for restic donation address, instead found this:

> It's the words that count most of all!

Ok, then:

I LOVE restic. It solved a decade-old issue for me in such an elegant way I can't stop being excited about it. So: *THANK YOU :heart:  to everyone involved in creating and maintaining restic. You rock and are heroes! :guitar: *

**Comment by Siilwyn** [comment]
Same, appreciate all the work! Restic saved my files a couple of times and it's been smooth sailing. :heart:

**Comment by skavoovie777** [comment]
I arrived at this close issue request after searching for a way to donate to the project. A collection of family photos was lost and once I discovered the hardware failure, I was safely restoring all 88 GiB of family photos to a new drive within minutes. Every picture recovered flawlessly. I can't express how grateful I am for all your work and the quality, usability, and features of Restic. Thank you!
